graylog2-server/src/main/java/org/graylog2/streams/OutputAVImpl.java

```diff
@@ -32,44 +32,52 @@
 @JsonAutoDetect
 @CollectionName("outputs")
 public abstract class OutputAVImpl implements Output {
+    static final String FIELD_ID = "_id";
```

Comment:
If this doesn't change the output format using the REST API, I'm fine. I just thought of #1248 in this context.